src/highlighter.js

```diff
@@ -594,18 +779,98 @@ export default function (options) {
         }
     }
 
+    /**
+     * @typedef HighlightEntryKeepEmpty
+     * @property {String} groupId
+     * @property {String} c - color
```

Comment:
Why you've chosen to use `c` intead of full name `color`?

Reply:
This name is used by another branch (`!keepEmptyNodes`), so for consistency - not that this consistency has any meaning now, since format is completely different. But let's keep it so we won't have to retest it :)